maximiliangarmatsch/maximiliangarmatsch.com
# reconfigure tests CI pipeline

Please provide a **brief description** of what this PR accomplishes.

This PR setup end-to-end test on Github Actions and Cypress Dashboard services. The PR also reconfigured the test suites by avoiding multiple test running on CI pipeline.



### How do I test this?

-   make a pull request 
-   see test suite on this link; [dashboard](https://dashboard.cypress.io/projects/cdbje3/runs?branches=%5B%5D&committers=%5B%5D&page=1&status=%5B%5D&tags=%5B%5D&timeRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D)

### Is there something controversial in your PR?
This PR deleted a Github Actions' workflow so has to avoid running multiple test suites that does the same thing. So this PR resorted to Cypress test suites that offers more functionalities like dashboard, test analytics etc.

### Did you remember to take care of the following?

-   [x] `npm install` – for new dependencies
